chia/cmds/chia.py

```diff
@@ -39,26 +42,58 @@ def monkey_patch_click() -> None:
     context_settings=CONTEXT_SETTINGS,
 )
 @click.option("--root-path", default=DEFAULT_ROOT_PATH, help="Config file root", type=click.Path(), show_default=True)
+@click.option(
+    "--keys-root-path", default=DEFAULT_KEYS_ROOT_PATH, help="Keyring file root", type=click.Path(), show_default=True
+)
+@click.option("--passphrase-file", type=click.File("r"), help="File or descriptor to read the keyring passphase from")
 @click.pass_context
-def cli(ctx: click.Context, root_path: str) -> None:
+def cli(ctx: click.Context, root_path: str, **kwargs) -> None:
     from pathlib import Path
 
     ctx.ensure_object(dict)
     ctx.obj["root_path"] = Path(root_path)
 
+    keys_root_path = kwargs.get("keys_root_path")
```

Comment:
Grabbing keys_root_path and passphrase_file from kwargs because when the feature is disabled, I scrub those options from Click. So they aren't guaranteed to be set.

Reply:
i would maybe add a small comment on that